Lib/difflib.py

```diff
@@ -1038,9 +1038,7 @@ def _qformat(self, aline, bline, atags, btags):
 # remaining is that perhaps it was really the case that " volatile"
 # was inserted after "private".  I can live with that <wink>.
 
-import re
-
-def IS_LINE_JUNK(line, pat=re.compile(r"\s*(?:#\s*)?$").match):
+def IS_LINE_JUNK(line, pat=None):
     r"""
     Return True for ignorable line: iff `line` is blank or contains a single '#'.
```

Comment:
```suggestion
    Return True for ignorable line: if `line` is blank or contains a single '#'.
```

there's also a typo in the docs, which can be fixed while we are at it.